wip/temp.jl

```diff
@@ -0,0 +1,34 @@
+using Mimi
```

Comment:
Was this file intended to be checked in? I can't see how it connects to the rest of the files in this PR

Reply:
Ah good point thank you, deleted it!